SwiftWisdom/Core/DirectoryManager/DirectoryManager.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 public final class DirectoryManager {
     
     enum Error : ErrorType {
-        case UnableToCreatePath(url: NSURL)
+        case UnableToCreatePath(url: NSURL?) // TODO: review, 2.3 changes
```

Comment:
In swift 2.2, there was never a case such that this error would occur but the url would not exist. However, in swift 2.3, there are more cases in which NSURL creation can fail resulting in an error. The existing UnableToCreatePath error was modified to capture this case rather than creating a new url-less error case.